yogstation/code/game/gamemodes/gangs/milestones.dm

```diff
@@ -0,0 +1,434 @@
+/datum/objective/gang  // Abstract
```

Comment:
don't actually think you need this since you don't change anything, BYOND does it automatically